pkgs/os-specific/linux/zfs/default.nix

```
@@ -226,7 +226,7 @@ in {
 
   zfsUnstable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.15";
+    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.16";
     latestCompatibleLinuxPackages = linuxPackages_5_14;
```

Instead of putting some made-up bogus values, just get rid of the check altogether. (in the same spirit as https://github.com/NixOS/nixpkgs/pull/145483)
```suggestion
    kernelCompatible = true;
    latestCompatibleLinuxPackages = pkgs.linuxPackages_latest;
```

---

Ideally, these should be updated when the version gets updated.

I don't see the harm if they are in sync